core/templates/dev/head/pages/exploration_editor/simple_editor_tab/SimpleEditorSidebarDirective.js

```diff
@@ -36,7 +39,7 @@ oppia.directive('simpleEditorSidebar', [
             'Multiple choice', 'Correct answer', 'Hints', 'Bridge text'];
           $scope.questionList = SimpleEditorManagerService.getQuestionList();
           $scope.ID_PREFIX = QuestionIdService.SIDEBAR_PREFIX;
-
+          $scope.modeSrv = SimpleEditorSidebarModeService;
```

Comment:
I have no idea what the varname here represents. I suggest:

    $scope.SimpleEditorSidebarModeService = SimpleEditorSidebarModeService